automatic/mkdocs/tools/ChocolateyInstall.ps1

```diff
@@ -1,10 +1,34 @@
 ﻿Update-SessionEnvironment
-
+$allowed_python_versions = @('3.9', '3.8', '3.7', '3.6', '3.5') # sync with nuspec
 $version = '1.1.2'
 
 $proxy = Get-EffectiveProxy
 if ($proxy) {
   Write-Host "Setting CLI proxy: $proxy"
   $env:http_proxy = $env:https_proxy = $proxy
 }
-python -m pip install mkdocs==$version
+
+function FindPython {
+  param(
+    $allowed_python_versions
+  )
+  # see https://www.python.org/dev/peps/pep-0514/#structure
+  # we are querying machine regsitry only because chocolatey
+  # installs python in admin mode only.
+  $avaiable_installation = Get-ChildItem -Path Registry::HKEY_LOCAL_MACHINE\Software\Python\PythonCore | Select-Object Name
```

Comment:
The variable `$avaiable_installation` is misspelt for the first part.

You don't need to use the `Registry::HKEY_LOCAL_MACHINE` - just use `HKLM:`